app/src/main/java/atlant/moviesapp/presenters/MovieListPresenter.java

```diff
@@ -82,6 +93,59 @@ public void onFailure(Call<MoviesResponse> call, Throwable t) {
 
     }
 
+    public void postFavorite(int id, String session_id, BodyFavourite favorite) {
+        final ApiInterface apiservice = ApiClient.getClient().create(ApiInterface.class);
```

Comment:
As mentioned before, please put the initialisation of the BodyFavourite object in this method